pkg/services/grafana-apiserver/service.go

```diff
@@ -180,55 +167,33 @@ func (s *service) start(ctx context.Context) error {
 		o.Etcd = nil
 	}
 
-	// Get the util to get the paths to pre-generated certs
-	certUtil := certgenerator.CertUtil{
```

Comment:
@charandas will there be any issues if we stop writing certs to disk?

Reply:
based on conversation with @charandas in the daily, this is not an issue. it was mainly used by authNZ hooks in previous PoC branches